src/styles/index.css

```diff
@@ -37,6 +37,10 @@ table {
   color: #535353;
 }
 
-.font-dark {
```

Comment:
Why you removed this?

Reply:
"@umeshcodewalnut, the CSS class was removed accidentally. I have now added it back. Could you please verify?"